platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -355,6 +356,69 @@ export default class DisplaySetService extends PubSubService {
         allDisplaySets.push(...displaySets);
       }
     }
+    // applying the default sopClassUID handler
+    if (allDisplaySets.length === 0) {
+      // applying hp-defined viewport settings to the displaysets
+      const defaultSOPClassHandler =
+        '@ohif/extension-default.sopClassHandlerModule.default';
```

Comment:
add a todo to make it customizable either from ExtensionManager or from `DisplaySetService.setUnsuportedSOPClassHandler`